common/url.go

```diff
@@ -315,6 +309,56 @@ func (c URL) Key() string {
 	//return c.ServiceKey()
 }
 
+func (c *URL) ColonSeparatedKey() string {
```

Comment:
maybe u can add a comment about this func name's origin [from dubbo, its filename], I think.

Reply:
i copy from this PR : [support sign and auth for request](https://github.com/apache/dubbo-go/pull/323/files) , but i can add some comment